src/Buttons/button.style.js

```diff
@@ -89,7 +89,7 @@ const ButtonStyles = styled.div`
 
     &:hover {
       ${transitionBoxShadow};
-      background-color: ${Colors.brandAlternative};
+      background-color: ${Colors.affirmative};
```

Comment:
This is a totally different colour? is it a mistake?

Reply:
Yes, all our buttons are green on hover now

![image](https://user-images.githubusercontent.com/492636/61124967-bf275e00-a4a8-11e9-8e40-cdb72989d50c.png)